test/functional/admin/useful_news_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class Admin::UsefulNewsControllerTest < ActionController::TestCase
+  setup do
+    @admin_useful_news = admin_useful_news(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:admin_useful_news)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create admin_useful_news" do
+    assert_difference('Admin::UsefulNews.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.